mitmproxy/contentviews/img/metadata.py

```diff
@@ -0,0 +1,30 @@
+from . import png
```

Comment:
Because `image.py` is also a file and running `mitmproxy` shows the error `AttributeError: module 'mitmproxy.contentviews.image' has no attribute 'ViewImage'`